modules/local/pretext_map.nf

```diff
@@ -0,0 +1,35 @@
+process PRETEXT_MAP {
```

Comment:
I think at the moment input of `pretext map` is cut that way so that it's tightly bound to the output of `juicer pre`. I could move it to a global module if that's what you need for your pipeline as well.

Reply:
To be more precise, the standard usage [here](https://github.com/wtsi-hpag/PretextMap) suggests something like that

`samtools view -h file.bam seq_1 seq_2 | PretextMap -o map.pretext --filterInclude "seq_1, seq_2"`

And from the command-line

PretextMap Version 0.1.9

  (...samformat, ...pairsformat |)  PretextMap -o output.pretext
                                                        (--sortby ({length}, name, nosort)
                                                        --sortorder ({descend}, ascend)
                                                        --mapq {10}
                                                        --filterInclude "seq_ [, seq_]*"
                                                        --filterExclude "seq_ [, seq_]*")
                                                        {--highRes}
  (< samfile, pairsfile)

which makes it depend on samtools basically. Should be implementable with the optional parameters (either samfile or pairsfile).